src/actions/public/simple-bitcoin-vault.ts

```diff
@@ -92,3 +97,25 @@ export async function getVaultStatus(
     functionName: "getStatus",
   });
 }
+
+export async function getVaultPendingWithdrawalsData(
+  client: Client,
+  parameters: { vaultAddress: Address },
+) {
+  assertObject(parameters, "parameters");
+  const { vaultAddress } = parameters;
+  assertAddress(vaultAddress, "vaultAddress");
+  const vaultStateAddress = await getBitcoinVaultStateAddress(client, {
+    vaultAddress,
+  });
+  const pendingWithdrawalCount = await getPendingWithdrawalCount(client, {
+    vaultStateAddress,
+  });
+  const pendingWithdrawalAmountSat = pendingWithdrawalCount
+    ? await getPendingWithdrawalAmountSat(client, { vaultStateAddress })
+    : BigInt(0);
+  return {
+    pendingWithdrawalAmountSat,
+    pendingWithdrawalCount,
+  };
+}
```

Comment:
I wonder if this should be in the consumers instead of on `hemi-viem`, as it's a combination of 2 functions from `hemi-viem` (which is ok that those are here). Thoughts?